pfrl/wrappers/atari_wrappers.py

```diff
@@ -37,9 +37,10 @@ def reset(self, **kwargs):
         if self.override_num_noops is not None:
             noops = self.override_num_noops
         else:
-            noops = self.unwrapped.np_random.randint(
-                1, self.noop_max + 1
-            )  # pylint: disable=E1101
+            if gym.__version__ >= "0.24.0":
```

Comment:
This way of comparing two versions can go wrong. If `gym.__version__` is `"0.3.0"`, `gym.__version__ >= "0.24.0"` is True. Can you use `StrictVersion` instead like https://github.com/pfnet/pfrl/blob/cee0777401fd3f8cf6b32d31337b88dcf7817059/examples/atlas/train_soft_actor_critic_atlas.py#L23-L24 ?

Reply:
It seems StrictVersion is a less modern/preferred way of doing things? 

https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

Should the `packaging` package be used instead?

Reply:
muupan: thank you for spotting this error.

Also I note Prabhat's comment regarding the use of the "Packaging" package as an alternative. From what I have read, I found this to be convincing as a more modern approach.

Muupan, as you work for Preferred Networks Inc., perhaps you could give guidance as to which is to be preferred?

Thanks.

Reply:
Thanks for pointing it out, Prabhat! I am convinced that `packaging` is better. `packaging` is a third party module, but it seems it is included in `setuptools`, which we already requires in `setup.py`.

@ jfrancis71 Can you fix it with `packaging`?